README.md

```diff
@@ -22,6 +22,19 @@ You can use the following table to learn more about each sample, and find the co
 | [High throughput](high-throughput) | Learn how you can design your smart contracts to process a large volume of transactions. | |
 | [Chaincode](chaincode) | A set of sample smart contracts used by other samples and the tutorials in the Fabric documentation. | [Fabric tutorials](https://hyperledger-fabric.readthedocs.io/en/master/tutorials.html) |
 
+### Asset transfer smart contract series
+
+The Asset transfer series provides a series of smart contracts and applications that you can use to manage and transfer assets using Hyperledger Fabric. Each smart contract creates a generic asset and can transfer the asset to new owner. However, each sample is built with different smart contract and application APIs to order to demonstrate different Fabric features. The **Basic** sample provides an introduction to how smart contracts work on Fabric, and how the Fabric SDK's can be used to build applications that can interact with Fabric networks. The **Secured agreement** sample demonstrates how to use more advanced capabilities.
```

Comment:
nit:
```suggestion
The Asset transfer series provides a series of smart contracts and applications that you can use to manage and transfer assets using Hyperledger Fabric. Each smart contract creates a generic asset and can transfer the asset to a new owner. However, each sample is built with different smart contract and application APIs in order to demonstrate different Fabric features. The **Basic** sample provides an introduction to how smart contracts works on Fabric, and how the Fabric SDK's can be used to build applications that can interact with Fabric networks. The **Secured agreement** sample demonstrates how to use more advanced capabilities.
```

Reply:
Tweeking this paragraph:  
The asset transfer series provides a series of smart contracts and applications that you can use to create and transfer a generic asset using Hyperledger Fabric. However, each sample is built with different smart contract and application APIs in order to demonstrate different Fabric features. The **Basic** sample provides an introduction on how to write smart contracts and how to interact with a Fabric network using the Fabric SDKs. The **Secured agreement** sample demonstrates how to use more advanced capabilities to develop a more realistic transfer scenario.